content/html/concepts/semantic-html/terms/ruby-text/ruby-text.md

```diff
@@ -0,0 +1,26 @@
+---
+Title: '<rt>'
+Description: 'Used to provide pronunciation, translation, or transliteration information for East Asian typography.'
```

Comment:
This description is a very close to a verbatim copy of the Mozilla documentation. I would suggest rephrasing this to differentiate it further. Also the example used below is the same as in the Mozilla version. All docs content must be original. Please revise with an example you have developed.